src/test/java/com/google/jenkins/plugins/computeengine/CleanLostNodesWorkTest.java

```diff
@@ -82,7 +82,7 @@ public void shouldNotCleanAnyInstance() throws Exception {
 
     getWorker().doRun();
     verify(client).listInstancesWithLabel(eq(TEST_PROJECT_ID), anyMap());
-    verifyZeroInteractions(client);
+    verifyNoMoreInteractions(client);
```

Comment:
`verifyZeroInteractions` is deprecated and should use `verifyNoInteractions`. However, that causes a test failure as it finds an interaction. 
Would `verifyNoMoreInteractions` be a more appropriate test?

Reply:
this looks correct to me, as you are verifying there has been an interaction before and you are checking that there are no other one!